dev/documentation/en/resources.md

```diff
@@ -23,4 +23,16 @@ A CLI script to [generate submissions for import](https://gist.github.com/NateWr
 
 ## Generate metrics
 
-A CLI tool to [generate fake metrics data](https://gist.github.com/NateWr/32322ca420a0abb37c035506dd51601d). Creates fake stats entries for each submission in your instance.
\ No newline at end of file
+A CLI tool to [generate fake metrics data](https://gist.github.com/NateWr/32322ca420a0abb37c035506dd51601d). Creates fake stats entries for each submission in your instance.
+
+## Remove a locale key
+
+A CLI tool to [remove a locale key from every locale](https://gist.github.com/NateWr/dd912a94e8809d91698e59f16fa96a5d). This can be useful when a locale key is no longer used and needs to be removed from every locale.
```

Comment:
Likewise, this can/maybe should be moved into the software repo. If nothing else, it saves me removing useful functions when grepping around for cleanup.

Reply:
(There are a bunch of gettext tools for .po file manipulation, but I've found them to be surprisingly unhelpful so far -- maybe I've just misunderstood their idiom...)

Reply:
:+1: Added in https://github.com/pkp/pkp-lib/commit/b00af6cdccbed0265bb17c1e1a6f0cef4cce61ce